src/nvim/os/shell.c

```diff
@@ -390,7 +390,7 @@ static bool out_data_decide_throttle(size_t size)
   static uint64_t   started     = 0;  // Start time of the current throttle.
   static size_t     received    = 0;  // Bytes observed since last throttle.
   static size_t     visit       = 0;  // "Pulse" count of the current throttle.
-  static char       pulse_msg[] = { ' ', ' ', ' ', '\0' };
+  static char       pulse_msg[] = { '.', ' ', ' ', '\0' };
```

Comment:
is this to help the screen test?

Reply:
initially, but should be reverted (with 4 states again).